packages/frontend-core/src/fetch/ViewV2Fetch.js

```diff
@@ -35,9 +35,32 @@ export default class ViewV2Fetch extends DataFetch {
   }
 
   async getData() {
-    const { datasource, limit, sortColumn, sortOrder, sortType, paginate } =
-      this.options
-    const { cursor, query } = get(this.store)
+    const {
+      datasource,
+      limit,
+      sortColumn,
+      sortOrder,
+      sortType,
+      paginate,
+      filter,
+    } = this.options
+    const { cursor, query, definition } = get(this.store)
+
+    // If sort params are not defined, update options to store the sorting
+    // params built in to this view. This ensures that we can accurately
+    // compare old and new sorting params and skip a redundant API call.
+    if (!sortColumn && definition.sort?.field) {
+      this.options.sortColumn = definition.sort.field
+      this.options.sortOrder = definition.sort.order
+    }
+
+    // If sort params are not defined, update options to store the sorting
```

Comment:
Very NAB: duplicate comment here I believe, probably applies to both cases it's pasted above but maybe a single comment above both does the same trick?

Reply:
Yep good catch, that's a dupe!